local/modules/Tinymce/Resources/js/tinymce/filemanager/config/config.php

```diff
@@ -69,47 +69,63 @@
 // Argh, url_site is not defined ?!
 if (empty($base_url)) {
     // A we did not used the router to access this dialog, we cannot use the URL class. Use the good old method.
-    $base_url = $request->getSchemeAndHttpHost() . preg_replace('!/tinymce/filemanager/dialog.php.*$!', '', $_SERVER['REQUEST_URI']);
+    $base_url = $request->getSchemeAndHttpHost().preg_replace('!/tinymce/filemanager/dialog.php.*$!', '',
+            $_SERVER['REQUEST_URI']);
```

Comment:
why only this parameter on a new line ? Put all parameters on a new line or none.